xarray/core/groupby.py

```diff
@@ -66,6 +67,52 @@ def _dummy_copy(xarray_obj):
         raise AssertionError
     return res
 
+def _is_one_or_none(obj):
+    return obj == 1 or obj is None
```

Comment:
Interesting! I never knew that. 
I always thought ints & strings should use `==`, but for -1 to 256 `is` is fine: http://stackoverflow.com/questions/2239737/is-it-better-to-use-is-or-for-number-comparison-in-python